src/components/EditEventPage.js

```diff
@@ -1,11 +1,34 @@
 import React from "react";
+import { connect } from "react-redux";
 import EventForm from "./EventForm";
+import { editEvent, removeEvent } from "../actions/events";
 
-const EditEventPage = () => (
-  <>
-    <EventForm />
-    <button>Save changes</button>
-  </>
-);
+class EditEventPage extends React.Component {
+  onSubmit = event => {
+    this.props.editEvent(this.props.event.id, event);
+    this.props.history.push("/");
```

Comment:
Consider using some generic action for redirecting which will handle `history` mutation.

Reply:
Discussed.